records_mover/db/redshift/unloader.py

```diff
@@ -47,7 +47,8 @@ def unload_to_s3_directory(self,
                                table: str,
                                unload_plan: RecordsUnloadPlan,
                                directory: RecordsDirectory) -> int:
-        logger.info(f"Starting Redshift unload to {directory.loc}...")
+        logger.info(f"Starting Redshift unload to {directory.loc} as "
+                    f"{unload_plan.records_format}...")
```

Comment:
While using this switch, I wanted to make sure it works, so logging what format I'm writing out was important!